Asio somehow forgot to include the cpp20 example directory in the release tarball
Issue: https://github.com/chriskohlhoff/asio/issues/1026

This patch can be used for compiler versions where C++20 isn't supported anyway.

Patches the configure script directly because the tarball also doesn't shot autogen.sh script

Author: micketeer@gmail.com
--- configure.orig	2022-05-13 13:36:13.707284039 +0200
+++ configure	2022-05-13 13:36:24.613131225 +0200
@@ -5766,7 +5766,7 @@
 fi
 
 
-ac_config_files="$ac_config_files Makefile include/Makefile src/Makefile src/tests/Makefile src/tests/properties/Makefile src/examples/cpp03/Makefile src/examples/cpp11/Makefile src/examples/cpp14/Makefile src/examples/cpp17/Makefile src/examples/cpp20/Makefile"
+ac_config_files="$ac_config_files Makefile include/Makefile src/Makefile src/tests/Makefile src/tests/properties/Makefile src/examples/cpp03/Makefile src/examples/cpp11/Makefile src/examples/cpp14/Makefile src/examples/cpp17/Makefile"
 
 cat >confcache <<\_ACEOF
 # This file is a shell script that caches the results of configure
@@ -6562,7 +6562,6 @@
     "src/examples/cpp11/Makefile") CONFIG_FILES="$CONFIG_FILES src/examples/cpp11/Makefile" ;;
     "src/examples/cpp14/Makefile") CONFIG_FILES="$CONFIG_FILES src/examples/cpp14/Makefile" ;;
     "src/examples/cpp17/Makefile") CONFIG_FILES="$CONFIG_FILES src/examples/cpp17/Makefile" ;;
-    "src/examples/cpp20/Makefile") CONFIG_FILES="$CONFIG_FILES src/examples/cpp20/Makefile" ;;
 
   *) as_fn_error $? "invalid argument: \`$ac_config_target'" "$LINENO" 5;;
   esac
